websites/D/Dribbble/dist/metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
```

Outdated schema version
```suggestion
  "$schema": "https://schemas.premid.app/metadata/1.5",
```